reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -362,6 +456,55 @@ def download_deeprec_resources(azure_container_url, data_path, remote_resource_n
     os.remove(os.path.join(data_path, remote_resource_name))
 
 
+def mrr_score(y_true, y_score):
+    """Computing mrr score metric.
+
+    Args:
+        y_true (obj): ground-truth labels.
```

Comment:
The input of these metrics is a numpy array? or a different object?

Reply:
I will update the docstring.